dubbo-config/dubbo-config-spring/src/test/java/org/apache/dubbo/config/spring/registry/nacos/nacos/NacosServiceNameTest.java

```diff
@@ -16,23 +16,29 @@
  */
 package org.apache.dubbo.config.spring.registry.nacos.nacos;
 
+import org.apache.dubbo.common.URL;
 import org.apache.dubbo.registry.nacos.NacosServiceName;
-
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
+import static org.apache.dubbo.common.constants.CommonConstants.*;
+import static org.apache.dubbo.common.constants.RegistryConstants.CATEGORY_KEY;
 import static org.apache.dubbo.common.constants.RegistryConstants.DEFAULT_CATEGORY;
+import static org.apache.dubbo.registry.nacos.NacosServiceName.DEFAULT_PARAM_VALUE;
 import static org.apache.dubbo.registry.nacos.NacosServiceName.WILDCARD;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
 
 /**
  * {@link NacosServiceName} Test
  *
  * @since 2.7.3
  */
-public class NacosServiceNameTest {
+class NacosServiceNameTest {
```

Comment:
why move `public`?

Reply:
Sorry, it is a mistake, I have restored it